index.html

```diff
@@ -622,6 +622,17 @@ <h2>
 <code>type</code>, and a <code>controller</code>, as well as other properties
 which depend on what type of key it is.
       </p>
+
+      <p>
+When expressing public key material in a DID Document, this specification
+strives to provide as few formats as possible to increase the likelihood of
+interoperability. The fewer formats that implementers have to implement, the
```

Comment:
```
When expressing public key material in a DID Document, this specification
strives to provide as few formats as possible to increase the likelihood of
interoperability.
```
->
```
This specification strives to limit the number of formats for expressing 
public key material in a DID Document to the fewest possible, to increase 
the likelihood of interoperability.
```

Reply:
Fixed in 8ae2a154f700647dfb3a9780e52d357269beb7b6.